frontend/src/components/eMIB/OrganizationalStructure.jsx

```diff
@@ -84,6 +88,10 @@ class OrganizationalStructure extends Component {
       this.setState({
         markdown_en: response.background.en.background[0].markdown[2].text,
         markdown_fr: response.background.fr.background[0].markdown[2].text,
+        popupMarkdownTitle_en: response.background.en.background[0].markdown[5].text,
```

Comment:
Again, a hybrid approach that will need to be revisited before pizza will fully work; ok for this PR

Reply:
hybrid in the sense that the markdowns are hard-coded